app/controllers/ItsaStatusController.scala

```diff
@@ -31,24 +31,35 @@ class ItsaStatusController @Inject()(cc: MessagesControllerComponents,
                                     (implicit val ec: ExecutionContext)
   extends FrontendController(cc) with Logging {
 
-  private def createOverwriteItsaStatusUrl(nino: String, taxYearRange: String): String = {
-    s"/income-tax/$nino/person-itd/itsa-status/$taxYearRange?futureYears=false&history=false"
+  private def createOverwriteItsaStatusUrl(nino: String, taxYear: TaxYear): String = {
+    s"/income-tax/$nino/person-itd/itsa-status/${taxYear.formattedTaxYearRange}?futureYears=false&history=false"
```

Comment:
we would have to rename formattedTaxYearRange method, as this line is a good example how we duplicating things with this naming convention, something taxYear.formetted or similar short naming would work fine